src/io.cpp

```diff
@@ -684,6 +695,28 @@ Pipeline SetPipeline(const PipelineOptions &values) {
         result.starIdAlgorithm = std::unique_ptr<StarIdAlgorithm>(new GeometricVotingStarIdAlgorithm(DegToRad(values.angularTolerance)));
     } else if (values.idAlgo == "py") {
         result.starIdAlgorithm = std::unique_ptr<StarIdAlgorithm>(new PyramidStarIdAlgorithm(DegToRad(values.angularTolerance), values.estimatedNumFalseStars, values.maxMismatchProb, 1000));
+    } else if (values.idAlgo == "tracking") {
+
+        if (values.prevAttitudeString == "") {
+            std::cout << "WARNING: information about the previous attitude has not been passed in while tracking mode is set." << std::endl;
+        }
+
+        // convert user inputted string ra,dec,roll to individual floats
+        std::vector<float> attInputs;
+        std::stringstream ss (values.prevAttitudeString);
```

Comment:
Taking attitude input as a single comma-separated string is a good idea, but we need to apply it consistently across the codebase. If we're going to do it here, we also need to replace the `--generate-ra`, `--generate-de`, and `--generate-roll` options with a single combined option (and there should be a separate function for parsing comma-separated attitudes, in attitude-utils.cpp)